server/planning/events.py

```diff
@@ -435,3 +441,14 @@ def generate_recurring_dates(start, frequency, interval=1, until=None, byday=Non
         return (tz.localize(dt).astimezone(pytz.UTC).replace(tzinfo=None) for dt in dates)
     else:
         return (date for date in dates)
+
+
+def setRecurringMode(event):
```

Comment:
I added here because this is an eve issue, when you update a dictionary actually eve is doing an merge.
Because of this we have to force some fields to be None in order to reset the current value.
Can be moved on client but I think it will be a little confusing for a front-end developer.